themes/10up-theme/includes/blocks.php

```diff
@@ -90,6 +92,45 @@ function blocks_editor_styles() {
 
 }
 
+/**
+ * Allowed blocks in editor
+ * See reference of available blocks here: https://wordpress.org/support/article/blocks/
+ *
+ * @param array $allowed_block_types core array
+ * @return array  $allowed_blocks modified array under right conditions.
+ */
+function allowed_block_types( $allowed_block_types ) {
+
+	$allowed_blocks = [
+		'core/paragraph',
+		'core/image',
```

Comment:
Also, alpha sort this list to easily check what is on this list? Took me a bit to look through the list to see if anything important like core/embed was missing.